client/src/pages/SearchCategoryPage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { CategorySection } from "@components/categories/CategorySection";
+import { ServiceSection } from "@components/services/ServiceSection";
+import styles from "./ServicesPage.module.scss";
+
+type Params = {
+  category: string;
+};
+
+export const SearchCategoryPage: React.FC = () => {
+  const { category } = useParams<Params>();
+
+  return (
+    <section
+      className={`container flex-row container-border-top ${styles.servicesContainer}`}
+    >
+      <CategorySection isVertical={true} />
+      {category && (
```

Comment:
O kas jei nebus kategorijos? Jei tarkim šitą komponentą užkrautumėt kitu URL ir jis nerastų parametro category?